package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@verdaccio/types",
-  "version": "2.0.1",
+  "version": "2.0.2",
```

Comment:
This was not required, usually is better to handle it with the command `npm version patch` but, ok.